docs/patterns/logo-wall/style.md

```diff
@@ -0,0 +1,109 @@
+</uxdot-example>
+
+1. Logo
+2. Border
+
+## Theme
+
+Logo walls can be used in light and dark themes.
+
+<rh-alert state="warning">
+  When using a borderless logo wall, ensure that the color contrast between each logo and the background meets accessibility standards.
+</rh-alert>
+
+### Light theme
+
+<uxdot-example width-adjustment="400px">
+    <img src="{{ '../style-theme-light.svg' | url }}" alt="Image of bordered and borderless logos against 3 light themed backgrounds.">
+</uxdot-example>
+
+### Dark theme
+
+<uxdot-example width-adjustment="600px">
+    <img src="{{ '../style-theme-dark.svg' | url }}" alt="Image of bordered and borderless logos against 4 dark themed backgrounds.">
+</uxdot-example>
+
+## Space
+
+Bordered:
+ - Logos are evenly spaced.
+ - Logos are in a container with 16px of padding on all sides and a corner radius of 3px.
+
+<uxdot-example width-adjustment="800px">
+    <img src="{{ '../style-space-group-bordered.png' | url }}" alt="Example of a bordered logo wall with spacing values.">
+</uxdot-example>
+
+Borderless:
+ - Logos are evenly spaced.
+ - Logos are in a container with 16px of top and bottom padding and 8px of left and right padding.
+ - All containers using the borderless style have the same height but will likely have different widths depending on the dimensions of the logo.
+
+<uxdot-example width-adjustment="800px">
+    <img src="{{ '../style-space-group-borderless.png' | url }}" alt="Example of a borderless logo wall with spacing values.">
+</uxdot-example>
+
+## Interaction states
+
+Interaction states are visual representations used to communicate the status of an element or pattern.
+
+### Hover
+
+<uxdot-example width-adjustment="400px">
+    <img src="{{ '../style-states-hover-light.svg' | url }}" alt="Image showing hover states in light theme. The bordered logo background changes to light gray. The borderless logo has a red border on the bottom.">
+</uxdot-example>
+
+<uxdot-example width-adjustment="400px">
+    <img src="{{ '../style-states-hover-dark.svg' | url }}" alt="Image showing hover states in dark theme. The bordered logo background changes to lighter gray. The borderless logo has a red border on the bottom.">
+</uxdot-example>
+
+### Focus
+
+<rh-alert state="info">
+    The Focus states have the same styles as the Hover states, except for the focus ring.
+</rh-alert>
+
+<uxdot-example width-adjustment="400px">
+    <img src="{{ '../style-states-focus-light.svg' | url }}" alt="Image showing focus states in light theme. The focus states are the same as the hover states with an added focus ring.">
+</uxdot-example>
+
+<uxdot-example width-adjustment="400px">
+    <img src="{{ '../style-states-focus-dark.svg' | url }}" alt="Image showing focus states in dark theme. The focus states are the same as the hover states with an added focus ring.">
+</uxdot-example>
+
+### Active
+
+<rh-alert state="info">
+    The Active states have the same styles as the Focus states.
```

Comment:
Should we capitalize active and focus? I copied this from the alert on the button style page. @markcaron  @marionnegp